### DISCUSSION TITLE
Stacking two datasets on y-axis

### DISCUSSION THREAD

**Discussion by LoganMcnulty** [TanStack/react-charts#216] [Q&A] [answered]
Hi Everyone,

React-charts is so light-weight and user-friendly, thanks for making it. 

Seeking help with a scaling inquiry. Trying to stack two datasets on top of each other. The first dataset scales from 0.00-100.00 (a score), and the next from theoretically 0.00-infinity (stock price). The two datasets often vary greatly in scale, affecting legibility of the graph with my current settings.

Have played around with the type parameters etc., but no luck finding my solution.

**Image example** - the score is in red, and price is in blue. 
![image](https://user-images.githubusercontent.com/52426539/125434279-250db2c8-3a23-4c96-9ca2-1bf536541e9d.png)

**Links to live examples**
[Example 1](https://trendedge.app/asset/googl)
[Example 1](https://trendedge.app/asset/GPRO)
[Example 1](https://trendedge.app/asset/ARKK)

Thoughts? And thank you so much in advance. 

Logan

**Comment by LoganMcnulty** [comment, marked as answer]
May have found my answer at this link

https://react-charts.js.org/examples/multiple-axes

**Reply by LoganMcnulty** [reply]
Having trouble with passing data in the correct format. Erroring at .map of data = React.useMemo = () => ...

**Reply by tannerlinsley** [reply]
One option here is to normalize the chart values into percentage of change over time, so they can be compared accurately. The next option is to use a dual-secondary axis.

**Reply by LoganMcnulty** [reply]
Thanks for inquiring. Have made progress since post, here's an update if you're interested.

Top graph = multi-axis (price and score are two separate axis on purpose bc score is an oscillator).
Bottom graph = multi-type (the area over the bars looks great!)

![image](https://user-images.githubusercontent.com/52426539/126038280-652fe8d9-63fb-4b6e-92d4-12ee00b5b86e.png)

Now going to move Trend Edge into volume graph, and add moving averages to price graph.

Thanks again, react-charts is awesome.

**Comment by mustafa-hanif** [comment]
> https://react-charts.js.org/examples/multiple-axes

The link is no longer valid, pleae help
